src/apps/explorer/pages/AppData/config.tsx

```diff
@@ -0,0 +1,290 @@
+import React, { RefObject } from 'react'
+import Form, { AjvError, FieldProps, FormValidation } from '@rjsf/core'
+import {
+  LATEST_APP_DATA_VERSION,
+  LATEST_QUOTE_METADATA_VERSION,
+  LATEST_REFERRER_METADATA_VERSION,
+  getAppDataSchema,
+} from '@cowprotocol/app-data'
+import { JSONSchema7 } from 'json-schema'
+import { HelpTooltip } from 'components/Tooltip'
+
+const ERROR_MESSAGES = {
+  REQUIRED: 'Required field.',
+  INVALID_ADDRESS: 'This is not an address.',
+  ONLY_DIGITS: 'Only digits are allowed.',
+  INVALID_APPDATA: 'This is not a valid AppData hash.',
+}
+
+export const INITIAL_FORM_VALUES = {
+  version: LATEST_APP_DATA_VERSION,
+  metadata: {},
+}
+
+export const INVALID_IPFS_CREDENTIALS = [
+  'Type error',
+  "Failed to execute 'setRequestHeader' on 'XMLHttpRequest': String contains non ISO-8859-1 code point.",
+]
+
+export type FormProps = Record<string, any>
+
+export const getSchema = async (): Promise<JSONSchema7> => {
+  const latestSchema = (await getAppDataSchema(LATEST_APP_DATA_VERSION)).default as JSONSchema7
+  deleteAllPropertiesByName(latestSchema, 'examples')
+  deleteAllPropertiesByName(latestSchema, '$id')
+  return formatSchema(latestSchema)
+}
+
+const formatSchema = (schema: JSONSchema7): JSONSchema7 => {
+  const formattedSchema = structuredClone(schema)
+
+  if (formattedSchema?.properties?.version) {
+    formattedSchema.properties.version['readOnly'] = true
+    formattedSchema.properties.version['default'] = LATEST_APP_DATA_VERSION
```

Comment:
Damn, this looks like the thing that SHOULD be in the schema definition.

If this is what you were pointing out all along Mati, I'm very sorry that I never understood your proposal.

I've created this issue to address that https://github.com/cowprotocol/app-data/issues/11